var/spack/repos/builtin/packages/amrex/package.py

```diff
@@ -121,6 +121,14 @@ def url_for_version(self, version):
             url = "https://github.com/AMReX-Codes/amrex/archive/{0}.tar.gz"
         return url.format(version.dotted)
 
+    def get_cuda_arch_string(self, values):
+        if 'none' in values:
+            return('Auto')
+        else:
+            # Use format x.y instead of CudaPackage xy format
```

Comment:
Oh wow, I looked at this and was sure xy is also what we pass. Well, we don't :)